### PR TITLE
Remove $ symbol from some commands

### PR DESCRIPTION
**Pull request by cdrage**
**What type of PR is this:**

/kind documentation

**What does this PR do / why we need it:**

This PR does the following:
* Removes all `$` from commands that should be "copied-and-pasted"
* Separates all console output to a separation section.

Notes:
* "Details" markdown feature would of been used, but Docusaurus does not
  support <details> within a <tab>.

**Which issue(s) this PR fixes:**

Fixes https://github.com/redhat-developer/odo/issues/5844

**PR acceptance criteria:**

- [X] Documentation

**How to test changes / Special notes to the reviewer:**

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

File: docs/website/docs/command-reference/create-namespace.md [hunk 1]
```diff
@@ -10,10 +10,7 @@ To create a namespace you can run `odo create namespace <name>`:
 ```shell
 odo create namespace mynamespace
 ```
-
-Example - 
 ```shell
-odo create namespace mynamespace
```
Comment (valaparthvi) [inline docs/website/docs/command-reference/create-namespace.md:16]: I am not sure about this. Currently, both the section about command and output seem unrelated to each other.
I would rather keep this as the following -
```sh
$ odo create namespace mynamespace
 ✓  Namespace "mynamespace" is ready for use
 ✓  New namespace created and now using namespace: mynamespace
```

Same comment for all the changes.

File: docs/website/docs/command-reference/create-namespace.md [hunk 2]
```diff
@@ -10,10 +10,7 @@ To create a namespace you can run `odo create namespace <name>`:
...
  ✓  Namespace "mynamespace" is ready for use
  ✓  New namespace created and now using namespace: mynamespace
 ```
```
Comment (valaparthvi) [inline docs/website/docs/command-reference/create-namespace.md:16]: I am not sure about this. Currently, both the section about command and output seem unrelated to each other.
I would rather keep this as the following -

-----
To create a namespace you can run `odo create namespace <name>`.
Example - 
```sh
$ odo create namespace mynamespace
 ✓  Namespace "mynamespace" is ready for use
 ✓  New namespace created and now using namespace: mynamespace
```

-----
Same comment for all the changes.

**Review by valaparthvi — CHANGES_REQUESTED** [review]

File: docs/website/docs/command-reference/create-namespace.md (hunk 2 above)
Comment (dharmit) [reply docs/website/docs/command-reference/create-namespace.md:16]: +1. I was scrolling through `odo describe binding` and it felt weird to me. Attaching screenshot:

image

Comment (cdrage) [reply docs/website/docs/command-reference/create-namespace.md:16]: Yup, that's intentional. But @valaparthvi has a good point and I'll update it so that it includes the commands for the examples.

The intention was to keep it separate so we don't have to add a "Example output:" each time to each command..

**Comment by cdrage** [comment]
Updated the PR based on the review 👍

File: docs/website/docs/command-reference/describe-binding.md [hunk 3]
```diff
@@ -98,10 +104,11 @@ Available binding information:
 
 ### Describe without access to Devfile
 
-```shell
+```console
 odo describe binding --name <component_name>
 ```
 
+$ odo describe binding --name <component_name>
```
Comment (valaparthvi) [inline docs/website/docs/command-reference/describe-binding.md:111]: This is not code blocked. Can you fix it?

**Review by valaparthvi — CHANGES_REQUESTED** [review]
This is probably out of scope for this PR, but I still think it's relevant to it.
There are a few docs where the code block also includes a comment, I don't like the idea of copying the comment when I copy the code, and I was wondering if we can remove the comment out of the code block from the following docs:

- [ ] https://deploy-preview-5979--odo-docusaurus-preview.netlify.app/docs/command-reference/add-binding#interactive-mode
- [ ] https://deploy-preview-5979--odo-docusaurus-preview.netlify.app/docs/command-reference/completion#bash

I also noticed that there is an inconsistency in these command-reference docs. Some commands show a sample output along with them, while some don't. I'm wondering if you can fix that as well.

- [ ] https://deploy-preview-5979--odo-docusaurus-preview.netlify.app/docs/command-reference/describe-component
- [ ] https://deploy-preview-5979--odo-docusaurus-preview.netlify.app/docs/command-reference/delete-component
- [ ] https://deploy-preview-5979--odo-docusaurus-preview.netlify.app/docs/command-reference/logs
- [ ] https://deploy-preview-5979--odo-docusaurus-preview.netlify.app/docs/command-reference/remove-binding

`odo logs` command does not have copy command at all like other docs, do you think we can fix that?

I understand these are a lot of changes, and some unrelated, so I wouldn't mind if they're taken up in a separate PR.

**Comment by cdrage** [comment]
The comments in the code blocks IMO are okay, but we can leave that to a cabal topic. It's similar to `kubectl` / kubernetes.io documentation on their code copy blocks. Makes sense to keep the comments "within" the copy and paste code.

With regards to some commands being inconsistent in example output, you are totally right, that's outside the scope of the PR (that you mentioned).

The docs team plans to do a refactor of our documentation so we'll have a lot more consistency then for examples!

**Comment by valaparthvi** [comment]
/lgtm
Thanks for your work on this, Charlie.
